FlowCrypt.xcodeproj/project.pbxproj

```diff
@@ -892,7 +892,7 @@
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
 				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
-				ENABLE_BITCODE = YES;
+				ENABLE_BITCODE = NO;
```

Comment:
Releases should definitely have bitcode enabled, unless you have reason to disable them - in which case please let me know.